reporter-web-app/src/models/WebAppOrtResult.js

```diff
@@ -199,7 +199,7 @@ class WebAppOrtResult {
             if (obj.scopes) {
                 const { scopes } = obj;
                 for (let i = 0, len = scopes.length; i < len; i++) {
-                    const webAppScope = new WebAppScope(scopes[i]);
+                    const webAppScope = new WebAppScope(scopes[i], this);
```

Comment:
commit-msg:

scopes excludes -> scope excludes
'will now' -> maybe better: now shows

The bottom sentence could be made more compact, e.g.:

This change builds on top of eb01824 and c12995c.